pyproject.toml

```diff
@@ -215,3 +215,51 @@ filterwarnings = [
     # https://github.com/aio-libs/aiohttp/issues/6905
     '''ignore:'cgi' is deprecated and slated for removal in Python 3.13:DeprecationWarning''',
 ]
+
+[tool.coverage.report]
+omit = [
+  "src/blib2to3/*",
+  "tests/data/*",
+  "*/site-packages/*",
+  ".tox/*"
+]
+
+[tool.coverage.run]
+relative_files = true
+
+[tool.mypy]
+python_version = "3.7"
+mypy_path = "src"
+show_column_numbers = true
+show_error_codes = true
+strict = true
+no_implicit_reexport = true
+warn_unreachable = true
+
+[[tool.mypy.overrides]]
+module="blib2to3.driver.*"
+ignore_missing_imports = true
+
+[[tool.mypy.overrides]]
+module="IPython.*"
+ignore_missing_imports = true
+
+[[tool.mypy.overrides]]
+module="colorama.*"
+ignore_missing_imports = true
+
+[[tool.mypy.overrides]]
+module="pathspec.*"
+ignore_missing_imports = true
+
+[[tool.mypy.overrides]]
+module="tokenize_rt.*"
+ignore_missing_imports = true
+
+[[tool.mypy.overrides]]
+module="uvloop.*"
+ignore_missing_imports = true
+
+[[tool.mypy.overrides]]
+module="_black_version.*"
+ignore_missing_imports = true
```

Comment:
you can put all those under one subtable with modules as list. see: [example](https://github.com/python-poetry/poetry/blob/c8e963e668af2f71ddce2053e575ae0427b363e6/pyproject.toml#L179)

Reply:
@Secrus thanks for the feedback. Addressed in 52dbc57f542e6aff419c4c22143fdbdfd040eaa8